### PR TITLE
Allow development of the SDK alongside QFieldCloud

### PR DESCRIPTION
**Pull request by suricactus**

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3813 Basemaps uploaded multiple times after synchronization](https://app.clickup.com/t/2192114/QF-3813)

**Review by boardend — APPROVED** [review]
Looks good to me

File: docker-app/qfieldcloud/core/views/files_views.py [hunk 1]
```diff
@@ -83,6 +83,8 @@ def get(self, request: Request, projectid: str) -> Response:
             path = PurePath(version.key)
             filename = str(path.relative_to(*path.parts[:3]))
             last_modified = version.last_modified.strftime("%d.%m.%Y %H:%M:%S %Z")
+            # NOTE ETag is a MD5. But for the multipart uploaded files, the MD5 is computed from the concatenation of the MD5s of each uploaded part.
+            # TODO make sure when file metadata is in the DB, this is a real md5sum of the current file.
```
[the remark this answers is not in the document]
Comment (suricactus) [reply docker-app/qfieldcloud/core/views/files_views.py:87]: I would prefer we keep the TODO and note for the future us. As for the task, check QF-2760

Comment (suricactus) [inline docker-app/qfieldcloud/core/views/files_views.py:87]:
```suggestion
            # TODO make sure when file metadata is in the DB (QF-2760), this is a real md5sum of the current file.
```

**Pull request merged**